README.md

```diff
@@ -1,5 +1,10 @@
 # What is Milan ? 
 
+<div align="center">
+    
+<img src="https://img.shields.io/github/repo-size/IAmTamal/Milan?style=for-the-badge" />
+<img src="https://img.shields.io/github/package-json/v/IAmTamal/Milan?style=for-the-badge" />
+</div>
```

Comment:
There's no gap after the `</div>` and the next line as a result the bold text is breaking.
Currently, it looks like this : 

![image](https://github.com/IAmTamal/Milan/assets/72851613/30727ad7-df23-4774-a978-49add6506f4c)


Please move the badges to the top : 

![image](https://github.com/IAmTamal/Milan/assets/72851613/cd507d68-e024-4bf2-bb90-7afe001c5351)